plugin/pkg/scheduler/factory/factory.go

```diff
@@ -320,9 +320,14 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 		return nil, err
 	}
 
+	predicateMetaProducer, err := f.GetPredicateMetadataProducer()
+	if err != nil {
+		return nil, err
+	}
+
 	f.Run()
 
-	algo := scheduler.NewGenericScheduler(f.schedulerCache, predicateFuncs, priorityMetaProducer, priorityConfigs, extenders)
+	algo := scheduler.NewGenericScheduler(f.schedulerCache, predicateFuncs, priorityMetaProducer, predicateMetaProducer, priorityConfigs, extenders)
```

Comment:
I would prefer to change the order: first predicate related, then priority-related. This reflects both: alphabetic order, as well as the order in which they are processed internally.